Maths/Fraction.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @function fraction
+ * @description This function returns the fraction of a float type number.
+ * @param {number} num - a float type number is expected, but an integer will also work.
+ * @param {number} accuracy - the accuracy of the fraction, the default is 6.
+ * @return {Array} - an array containing the numerator and denominator of the fraction.
+ * @see https://en.wikipedia.org/wiki/Repeating_decimal and 
+ * @see https://en.wikipedia.org/wiki/Decimal#Decimal_fractions
+ * @example fraction(0.5) // [1, 2]
+ * @example fraction(0.3333333333333333) // [1, 3]
+ * @example fraction(0.25) // [1, 4]
+ * @example fraction(5.56) // [139, 25]
+ * @example fraction(0.33) // [33, 100]
+ */
+function fraction(number, accuracy = 6) {
+  let inp = typeof number === "number" ? number : false;
+  let acc = typeof accuracy === "number" ? accuracy : false;
+  if (inp !== false && acc !== false) {
+    if (Number.isInteger(inp)) {
+      return [inp, 1];
+    } else {
```

Comment:
This `else` is unneded due to the `return`.

Reply:
no, @appgurueu look closely. Here if the input is an integer, most of the case which is not, will return [inp,1] else it will do it's real job. It will not stop in if, if the number is not an integer.

Reply:
I did not ask you to resolve this by making the `else` "necessary" by removing the early return and replacing it with assignment to a return value. That just makes the code even harder to read.

Reply:
here else is "necessary"! Because if the input is not a integer type number, then what will happen!